dashboard/app/controllers/api/v1/pd/workshop_attendance_controller.rb

```diff
@@ -44,7 +44,9 @@ def update
       no_longer_attending = existing_user_ids - attending_user_ids
 
       new_attendees.each do |user_id|
-        Pd::Attendance.create session: session, teacher: User.find_by_id!(user_id)
+        Retryable.retryable(tries: 2, on: ActiveRecord::RecordNotUnique) do
+          Pd::Attendance.find_or_create_by! session: session, teacher: User.find_by_id!(user_id)
+        end
```

Comment:
Any reason that we explicitly specify the number of tries here, whereas typically (https://github.com/code-dot-org/code-dot-org/search?utf8=%E2%9C%93&q=Retryable.retryable&type=Code) we do not?